modules/cirrus/metrics/main.tf

```diff
@@ -0,0 +1,78 @@
+# CloudWatch Log Group
+resource "aws_cloudwatch_log_group" "workflow_events" {
+  name              = "${var.resource_prefix}-workflow-event-metrics"
+  retention_in_days = 14
```

Comment:
This probably needs to be an input? Can do in a followon PR to keep from adding more to this one if we want